p/scripts/user.query.js

```diff
@@ -11,27 +11,63 @@ function init_draggable() {
 		return;
 	}
 	
-	let source;
+	var source;
 	const configureQueries = document.querySelector('#configureQueries');
+	const addMarker = function(position, element) {
+		const hr = configureQueries.querySelector('hr.drag-drop-marker');
+		if (null === hr) {
+			element.insertAdjacentHTML(position, '<hr class="drag-drop-marker"/>');
+		}
+	}
+	const removeMarker = function() {
+		const hr = configureQueries.querySelector('hr.drag-drop-marker');
+		if (null !== hr) {
+			hr.remove();
+		}
+	}
 
 	configureQueries.addEventListener('dragstart', event => {
 		source = event.target.closest('[draggable="true"]');
 		event.dataTransfer.setData('text/html', source.outerHTML);
 		event.dataTransfer.effectAllowed = 'move';
 	});
-	configureQueries.addEventListener('dragover', event => event.preventDefault());
-	configureQueries.addEventListener('dragleave', event => event.preventDefault());
+	configureQueries.addEventListener('dragover', event => {
+		event.preventDefault();
+		if ('function' !== typeof event.target.closest) {
```

Comment:
No need to change it, but testing simply testing `if (event.target.closest)` is probably enough